### ISSUE TITLE
Error launching Setup: name 'sys' is not defined

### ISSUE THREAD

**Issue by VincentFrangi** [yorikvanhavre/BIM_Workbench#50]
I occurred an error launching BIM Setup

```
Running the Python command 'BIM_Setup' failed:
Traceback (most recent call last):
  File "/home/vincent/.FreeCAD/Mod/BIM/BimSetup.py", line 73, in Activated
    if sys.version_info.major < 3:

name 'sys' is not defined
```

OS: Manjaro Linux (i3-with-shmlog)
Word size of OS: 64-bit
Word size of FreeCAD: 64-bit
Version: 0.19.21509 (Git)
Build type: Release
Branch: master
Hash: ae641dc5278efafa22b168fc196875558b92e436
Python version: 3.8.3
Qt version: 5.14.2
Coin version: 4.0.0
OCC version: 7.4.0
Locale: English/United States (en_US)

**Comment by VincentFrangi** [comment]
I solve it by importing sys in BimSetup.py file

```python
import sys
```

**Comment by yorikvanhavre** [comment]
Fixed in 92619f5
Thanks for notifying!
